app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -190,7 +191,7 @@ class PrescriptionRepositoryAndroidTest {
     // then
     val expectedPrescribedDrug = prescribedDrug.copy(
         frequency = BD,
-        updatedAt = prescribedDrug.updatedAt.plus(durationToAdvanceBy),
+        timestamps = Timestamps(prescribedDrug.timestamps.createdAt, prescribedDrug.timestamps.updatedAt.plus(durationToAdvanceBy), prescribedDrug.timestamps.deletedAt),
```

Comment:
Same comment as above, We can use the `prescribedDrug.timestamps.copy` to update the `updatedAt` only.